dir.props

```diff
@@ -68,20 +68,25 @@
   <PropertyGroup>
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
+    <CoreFxExpectedPrerelease>rc3-23921</CoreFxExpectedPrerelease>
```

Comment:
Please update this to match the current value in dir.props before this gets merged.